src/core/facade/test_injector.ts

```diff
@@ -1,14 +1,16 @@
 import {Injector, Provider, PLATFORM_INITIALIZER} from 'angular2/core';
-import {BaseException, ExceptionHandler} from 'angular2/src/facade/exceptions';
+import {BaseException} from 'angular2/src/facade/exceptions';
 import {ListWrapper} from 'angular2/src/facade/collection';
 import {FunctionWrapper, isPresent, Type} from 'angular2/src/facade/lang';
 
 export class TestInjector {
+  /* tslint:disable:variable-name */
```

Comment:
I think we should fix the names instead of disabling the tslint.  This is just a matter of moving the `_` at the end, and removing it from the var.